app/controllers/users_controller.rb

```diff
@@ -1,2 +1,9 @@
 class UsersController < ApplicationController
+
+  def show
+    videos = @current_user.videos.to_ary.sort_by{|video| video.posted_at}.reverse![0..8]
+    tweets = @current_user.tweets.to_ary.sort_by{|tweet| tweet.posted_at}.reverse![0..8]
```

Comment:
Chaining methods can be really useful! But it can also become very confusing when overdone. Rule of thumb (from Sandi Metz): stop a chain if the type/class of the object changes. By this guideline, you'd do something like: 
```ruby
videos_array = @current_user.videos.to_ary
sorted_videos = videos_array.sort_by{|video| video.posted_at}.reverse![0..8]
```

Also, is `@current_user.videos` already an array?